cmake/compiler_config.cmake

```diff
@@ -107,4 +107,10 @@ function(mayaUsd_compile_config TARGET)
                 ${MSVC_DEFINITIONS}
         )
     endif()
+
+    # Remove annoying TBB warnings.
+    target_compile_definitions(${TARGET}
+        PRIVATE
+            TBB_SUPPRESS_DEPRECATED_MESSAGES
+    )
```

Comment:
I got tired of seeing hundreds of TBB warnings polluting our build log. This started with the update of TBB in Maya.